src/components/Footer/index.tsx

```diff
@@ -1,7 +1,43 @@
-import React from "react";
+import "./index.scss";
 
 const Footer = () => {
-  return <div></div>;
+  return (
+  <>
+ <div className="footer">
+      <ul className="footer__list">
+        <li className="footer__item pink">
+          Ada
+          <a href="https://www.linkedin.com/in/ada" className="footer__link pink">LinkedIn</a>
+        </li>
+        <li className="footer__item blue">
+          Andrzej
+          <a href="https://www.linkedin.com/in/andrzej" className="footer__link blue">LinkedIn</a>
+        </li>
+        <li className="footer__item pink">
+          Kunegunda
+          <a href="https://www.linkedin.com/in/kunegunda" className="footer__link pink">LinkedIn</a>
+        </li>
+        <li className="footer__item blue">
+          Pawel
+          <a href="https://www.linkedin.com/in/pawel" className="footer__link blue">LinkedIn</a>
+        </li>
+        <li className="footer__item pink">
+          Milena
+          <a href="https://www.linkedin.com/in/milena" className="footer__link pink">LinkedIn</a>
+        </li>
+        <li className="footer__item blue">
+          Simon
+          <a href="https://www.linkedin.com/in/simon" className="footer__link blue">LinkedIn</a>
+        </li>
+        <li className="footer__item pink">
+          Sandra
+          <a href="https://www.linkedin.com/in/sandra" className="footer__link pink">LinkedIn</a>
+        </li>
+      </ul>
+      <h1>Copyright 2024</h1>
```

Comment:
h1 is a main header on site, it will shown in google search so we should attached it in footer